src/components/timestamp-diff-util.jsx

```diff
@@ -0,0 +1,202 @@
+import _ from 'lodash';
+import * as xpath from 'simple-xpath-position';
+
+const absoluteUrlRegex = new RegExp(/\/\/web\.archive\.org\/web\/\d{14}/gm);
+const relativeUrlRegex = new RegExp(window.location.origin+ '/web/\\d{14}', 'gm');
+
+/**
+ * This function will parse all of the markup added by web-monitoring-processing in order to remove any false positive
+ * markup information (for example when comparing links to the same destination but with different timestamps
+ */
+export function getTimestampCleanDiff(insertions, deletions) {
+
+  //Create new HTML DOM elements to add web-monitoring-processing's responses to
+
+  let domIns = document.createElement( 'html' );
+  let domDel = document.createElement( 'html' );
+
+  domIns.innerHTML = insertions;
+  domDel.innerHTML = deletions;
+
+  let ins = domIns.getElementsByTagName('ins');
+  let del = domDel.getElementsByTagName('del');
+  let foundIns = [];
+  let foundDel = [];
+
+  //Get all of web-monitoring-processing's del elements that link to a resource and have at least one child,
+  // meaning that they highlight content
+  for(let i=0; i<del.length; i++) {
+    if (_.isEqual(del[i].className, 'wm-diff') && del[i].childNodes.length > 0) {
+      del[i].childNodes.forEach(function (child) {
+        const result = checkTimestampInLink(child);
+        //constant result might be nil, so we wouldn't want to add nill values to the array
+        addNotNill(foundDel, result);
+      });
+    }
+  }
+  //Get all of web-monitoring-processing's ins elements that link to a resource and have at least one child,
+  // meaning that they highlight content
+  for(let i=0; i<ins.length; i++) {
```

Comment:
Since this is a busy loop, its worth making it faster.
The following is faster because `ins.length` is evaluated only once and not in each loop.
```
for (let i = 0, len = ins.length; i < len; i++) {
```
Please do that in all busy loops if possible to make them faster.